scripts/build-traceviewer-module.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Usage:
+ *   node scripts/convert.js
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const jsdom = require('jsdom');
+const mkdirp = require('mkdirp');
+
+const paths = {};
+const INITIAL_IMPORT = 'scripts/traceviewer-module-index';
+
+function convertImport(src) {
+  console.log('Reading:', src);
+  const html = fs.readFileSync(src);
+  let dest = src.replace(/\.html$/, '.js');
+  dest = dest.replace(INITIAL_IMPORT, 'index');
+
+  jsdom.env({
+    html: html,
+    done: function(err, window) {
+      if (err) {
+        throw err;
+      }
+
+      const imports = window.document.querySelectorAll('link[rel="import"]');
+      const scripts = window.document.querySelectorAll('script');
+      let scriptsContent = '';
+
+      convertLicenseComments(html);
```

Comment:
We need some call to take across the LICENSE file from catapult and put it in the third_party folder, too.